.github/workflows/test_build_and_deploy.yml

```diff
@@ -40,9 +41,16 @@ jobs:
           hosts: ${{ secrets.SANDBOX_HOST }}
           privateKey: ${{ secrets.SSH_PRIV_KEY }}
           command: |
-            docker rm $(docker stop $(docker ps -a -q --filter ancestor=main_backend --format="{{.ID}}"))
-            docker rm $(docker stop $(docker ps -a -q --filter ancestor=main_frontend --format="{{.ID}}"))
-            docker rm $(docker stop $(docker ps -a -q --filter ancestor=ghcr.io/jellyfish-dev/jellyfish:edge --format="{{.ID}}"))
+            docker ps -a -q --filter ancestor=main_backend --format="{{.ID}}" | xargs -r docker stop
+            docker ps -a -q --filter ancestor=main_backend --format="{{.ID}}" | xargs -r docker rm
+            docker ps -a -q --filter ancestor=main_frontend --format="{{.ID}}" | xargs -r docker stop  
+            docker ps -a -q --filter ancestor=main_frontend --format="{{.ID}}" | xargs -r docker rm 
+            docker ps -a -q --filter ancestor=main_grafana --format="{{.ID}}" | xargs -r docker stop 
+            docker ps -a -q --filter ancestor=main_grafana --format="{{.ID}}" | xargs -r docker rm 
+            docker ps -a -q --filter ancestor=main_prometheus --format="{{.ID}}" | xargs -r docker stop  
+            docker ps -a -q --filter ancestor=main_prometheus --format="{{.ID}}" | xargs -r docker rm 
+            docker ps -a -q --filter ancestor=ghcr.io/jellyfish-dev/jellyfish:edge --format="{{.ID}}" | xargs -r docker stop 
+            docker ps -a -q --filter ancestor=ghcr.io/jellyfish-dev/jellyfish:edge --format="{{.ID}}" | xargs -r docker rm 
```

Comment:
This seems like a lot of work to remove 5 containers with known names. Maybe it would be even nicer to simply remove all of the containers, I suppose we only use the server for this deployment? Not sure tho, you can ignore this commend if you wish.